hugr/src/extension/op_def.rs

```diff
@@ -495,7 +502,10 @@ mod test {
         let type_scheme = PolyFuncType::new(vec![TP], FunctionType::new_endo(vec![list_of_var]));
 
         let def = e.add_op(OP_NAME, "desc".into(), type_scheme)?;
-        def.add_lower_func(LowerFunc::FixedHugr(ExtensionSet::new(), Hugr::default()));
+        def.add_lower_func(LowerFunc::FixedHugr {
+            extensions: ExtensionSet::new(),
+            hugr: Hugr::default(),
```

Comment:
If this is the test I should look at - no this doesn't look good! This is a Hugr that we can splice in in place of an Op (Node), so should be a DFG-rooted thing with the same inputs/outputs as specified in the OpDef's type_scheme....